File: APIdoc/php/tutorials/listFoldersDownloadFile.php
$email = "";
$password = "";
$apiKey = "";
$sessionKey = auth($email, $password, $apiKey);

$rules = list_access_rules($sessionKey)["RESULT"]["DATA"];
$folders = $list_folders($sessionKey, $rules[2]);
foreach($folders as $f){	
	if($f[2] == "FILE"){		
		$download = file_download(sessionKey, $f[16], $f[0], $f[1]);
		echo $download;
	}
}

File: APIdoc/python/tutorials/listFoldersDownloadFile.py
email, password, apiKey = "", "" ,""

sessionKey = pio_auth(email, password, apiKey)

accessRules = pio_list_access_rules(sessionKey)

folders = pio_list_folders(sessionKey, accessRules["RESULT"]["DATA"][2][1])["RESULT"]["DATA"]

for f in folders:
	if(f[2] == "FILE"):
		download = pio_file_download(sessionKey, f[16], f[0], f[1])
		print(download)